<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Options</class>
 <widget class="QDialog" name="Options">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>359</width>
    <height>196</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>359</width>
    <height>196</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title">
        <string>Set Mode</string>
       </property>
       <widget class="QRadioButton" name="p2cRadio">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>30</y>
          <width>125</width>
          <height>26</height>
         </rect>
        </property>
        <property name="text">
         <string>Player VS CPU</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="p2pRadio">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>60</y>
          <width>135</width>
          <height>26</height>
         </rect>
        </property>
        <property name="text">
         <string>Player VS Player</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="c2cRadio">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>90</y>
          <width>112</width>
          <height>26</height>
         </rect>
        </property>
        <property name="text">
         <string>CPU VS CPU</string>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title">
        <string>Set turn</string>
       </property>
       <widget class="QRadioButton" name="playerORadio">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>30</y>
          <width>112</width>
          <height>26</height>
         </rect>
        </property>
        <property name="text">
         <string>CPU</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="playerXRadio">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>60</y>
          <width>112</width>
          <height>26</height>
         </rect>
        </property>
        <property name="text">
         <string>Player</string>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>151</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <slots>
  <slot>setChanges()</slot>
 </slots>
</ui>
